core/src/main/java/org/jdbi/v3/SqlStatement.java

```diff
@@ -46,9 +46,9 @@
  * <code>Update</code>. It defines most of the argument binding functions
  * used by its subclasses.
  */
-public abstract class SQLStatement<SelfType extends SQLStatement<SelfType>> extends BaseStatement
+public abstract class SqlStatement<SelfType extends SqlStatement<SelfType>> extends BaseStatement
```

Comment:
What would you think of standardizing all self-return generic parameters like this:

    class Whatever<This extends Whatever<This>> {
      public This doSomething() {
        ...
        return (This) this;
      }
    }